### DISCUSSION TITLE
Does it work on nano pie?

### DISCUSSION THREAD

**Discussion by mehr32** [gpiozero/gpiozero#1158] [General]
Hello, this library is very interesting. Does it work on nano pie?

**Comment by lurch** [comment]
See https://github.com/gpiozero/gpiozero/issues/1022#issuecomment-1020622122 and https://github.com/gpiozero/gpiozero/discussions/1069

**Comment by mehr32** [comment]
It's not a very convincing excuse. Of course, it's your project, and you have the right to have your own policy, but other devices are worth it :) The only library that correctly worked on NanoPi based on my tests was python-periphery. Perhaps it would be good to take a look at it in the future if you decide otherwise. Anyway, I appreciate that you are developing this project.
